### ISSUE TITLE
Missing. Content type headers in POST HTTP transactions.

### ISSUE THREAD

**Issue by Frankenslag** [Hyrules/WinHue#218]
Apologies for not filling in the hue hub versions but I am trying to write a software emulated hub so It wouldn’t make sense.

I have notices that when WinHue POSTs  to a hub it doesn’t set a content type header. I assume that it should be application/json. 

I can work around this but I wonder if it should be set as a matter of course. I am not sure what a real hub expects in BTW.

**Comment by Hyrules** [comment]
I don't believe the content type header is mandatory. If you just send it the json content with the information it works as expected. I can add this no problem. This will not impact anything on my side.

**Comment by Frankenslag** [comment]
Thanks Hyrules. I would have added a pull request but not that practiced with github

**Comment by Hyrules** [comment]
No problem I have already fixed it so no need to add a pull request.

**Comment by Hyrules** [comment]
This has been implemented. Let me know if there is anything else.

**Comment by Hyrules** [comment]
By the way if you need any help testing your app I would be happy to help you testing it with WinHue.

**Comment by Frankenslag** [comment]
That’s O.K. I was trying to implement a very minimal hub implementation. i.e. just enough to get Amazon Echo to talk with my application and I believe it works now.

Initially I was using WinHue rather than my Echo to trigger the application but once I got that working then I stripped it down to a very ‘bare bones’ implementation running under Windows IoTCore on a RaspberryPi and only really capable of talking to the Echo properly.

Thanks for all your help.

Andrew Gould
